packages/material-ui-lab/src/TimelineSeparator/TimelineSeparator.d.ts

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react';
 import { InternalStandardProps as StandardProps } from '@material-ui/core';
+import { Theme } from '@material-ui/styles';
```

Comment:
`@material-ui/styles` is a legacy package. How about we do a follow-up with?

```suggestion
import { Theme } from '@material-ui/core/styles';
```

Reply:
@oliviertassinari @mnajdova Shall we raise a new PR? so we make that change also in the `TabPanel` that I saw that it was migrated in the same way.

Reply:
Yep, let’s do it!

Reply:
Done in #25682